plugins/modules.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const supported = {
+	commonjs: 'babel-plugin-transform-es2015-modules-commonjs'
+};
+
+module.exports = function (options) {
+	const modules = options && options.modules !== null ? options.modules : 'commonjs';
+
+	if (modules === false) {
+		return [];
+	}
+
+	const plugin = supported[modules];
+
+	if (!plugin) {
+		throw new Error('@ava/stage-4 only supports commonjs module transformation.');
+	}
+
+	return [plugin];
```

Comment:
I'd say this is clearer:

```js
if (!options || options.modules === false) {
    return [];
}

if (options.modules !== 'commonjs') {
     throw new Error('@ava/stage-4 only supports commonjs module transformation.');
}

return ['babel-plugin-transform-es2015-modules-commonjs'];
```

Reply:
Does babel always provides options, even when the preset is set without any?

Reply:
Ah no, my bad. We should default to `commonjs` if there are no options.